src/NServiceBus.RabbitMQ/AssemblyInfo.cs

```diff
@@ -1,9 +1,3 @@
-﻿using System.Reflection;
-using System.Runtime.InteropServices;
+﻿using System.Runtime.InteropServices;
 
-[assembly: AssemblyTitle("NServiceBus.RabbitMQ")]
-[assembly: AssemblyCopyright("Copyright NServiceBus. All rights reserved")]
-[assembly: AssemblyProduct("NServiceBus.RabbitMQ")]
-[assembly: AssemblyCompany("NServiceBus Ltd.")]
-[assembly: AssemblyConfiguration("release")]
```

Comment:
This is going to sound really stupid, but because this is no longer explicitly specified, the casing in the output assembly is different:

```C#
// from ILSpy
[assembly: AssemblyConfiguration("Release")]
```

Does that matter?